test/support/connection_test_helper.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Janus.ConnectionTest.Helper do
```

Comment:
I would add some documentation here on how it works. Also since this works only with fake handle.